.github/workflows/notebook-weekly-test.yml

```diff
@@ -0,0 +1,81 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+name: Test notebooks that submit jobs
+on:
+  schedule:
+    - cron: "0 0 * * 1" # At 00:00 on Monday
```

Comment:
We were thinking of every other week or even every month (probably too infrequent). Simply to conserve resources. The downside is it taking longer to catch problems.

Maybe we go with every two weeks?

Reply:
Sounds good, done :)